packages/neuron-ui/.eslintrc.js

```diff
@@ -26,9 +26,7 @@ module.exports = {
       "exports": "always-multiline",
       "functions": "ignore"
     }],
-    "import/no-extraneous-dependencies": [2, {
-      "devDependencies": true
-    }],
+    "import/no-extraneous-dependencies": "off",
```

Comment:
It's strongly recommended to turn this on

Reply:
If we turn this on, we can't introduce electron in the render process because the render process has no dependency on electron.

Reply:
How about turning it off in specific modules.